api/camunda/camunda-openapi.yaml

```
@@ -1131,7 +1252,7 @@ paths:
                 $ref: "#/components/schemas/IncidentSearchQueryResponse"
         "400":
           description: >
-            The incident search query failed.
+            The Incident Search Query failed.
```

```suggestion
            The Incident Search Query failed.
```

category: casing

resolved in https://github.com/camunda/camunda/pull/22037